forge-gui/res/cardsfolder/t/the_crowd_goes_wild.txt

```diff
@@ -0,0 +1,9 @@
+Name:The Crowd Goes Wild
+ManaCost:X G
+Types:Sorcery
+K:Assist
+A:SP$ PutCounter | CounterType$ P1P1 | TargetMin$ 0 | TargetMax$ X | ValidTgts$ Creature | TgtPrompt$ Select up to X target creatures | SubAbility$ DBTrample | SpellDescription$ Support X. (Put a +1/+1 counter on each of up to X target creatures.)
+SVar:DBTrample:DB$ PumpAll | ValidCards$ Creature.counters_GE1_P1P1 | KW$ Trample | SpellDescription$ Each creature with a +1/+1 counter on it gains trample until end of turn.
```

Comment:
@Northmoc how should I correctly display the Oracle on these lines? It currently shows like this but the card images has them on new lines:

![image](https://github.com/Card-Forge/forge/assets/18243520/10ca18d7-05b2-4517-9b4a-713aecd74fd2)

Reply:
You could try putting a \n before the word Each, but also its not something we really worry about consistently.

Reply:
\n didn't work for that line but if it's not a big deal we can merge it like this and sort it another time